penaltymodel/core/interface.py

```diff
@@ -105,8 +119,12 @@ def penaltymodel_factory(priority):
         105
 
     """
+    warnings.warn(
+        "penaltymodel_factory() is deprecated and will be removed in penaltymodel 1.2.0",
```

Comment:
@randomir given our discussion the other day, I will change all of these to say "removed in penaltymodel 2.0.0"